boards/arm/mimxrt1060_evk/mimxrt1060_evk_defconfig

```diff
@@ -14,3 +14,6 @@ CONFIG_CORTEX_M_SYSTICK=y
 CONFIG_GPIO=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=600000000
 CONFIG_ARM_MPU=y
+CONFIG_DMA=y
+CONFIG_DMA_IMX_RT=y
+CONFIG_NOCACHE_MEMORY=y
```

Comment:
Instead of adding these lines to the board defconfig, update `soc/arm/nxp_imx/rt/Kconfig.defconfig.series` to enable the driver. Something like:
```
config DMA_MCUX_EDMA
        default y if HAS_MCUX_EDMA
        depends on DMA
```

Reply:
ok

Reply:
done